rust/agents/relayer/src/msg/serial_submitter.rs

```diff
@@ -193,11 +193,11 @@ impl SerialSubmitter {
             None => return Ok(()),
         };
 
-        if msg.num_retries >= 16 {
+        if msg.num_retries >= 1 {
             let required_duration = Duration::from_secs(match msg.num_retries {
-                i if i < 16 => unreachable!(),
-                // wait 5 min
-                i if (16..24).contains(&i) => 60 * 5,
+                i if i < 1 => unreachable!(),
+                // wait 20s to 7.7min with a linear increase
+                i if (1..24).contains(&i) => i as u64 * 20,
```

Comment:
```suggestion
                // wait 10s
                i if (1..12).contains(&i) => 10,
                // wait 20s to 7.7min with a linear increase
                i if (12..24).contains(&i) => i as u64 * 20,
```

Obviously this is more of an art than a science, but what do you think about this maybe? I would still love to maintain responsiveness for the fast finality chains as much as we can and not drop down to 20/40/60s if validators are just slightly slow

Reply:
Went ahead and took the flat 10 for the first few and then altered the linear scaling to make more sense between it and the exponential backoff.